tests/publiccloud/transfer_repos.pm

```diff
@@ -57,7 +57,7 @@ sub run {
         # Mitigate occasional CSP network problems (especially one CSP is prone to those issues!)
         # Delay of 2 minutes between the tries to give their network some time to recover after a failure
         # For rsync the ~/repos/./ means that the --relative will take efect after.
-        script_retry("rsync --timeout=$timeout -uahRd -e ssh --files-from /tmp/transfer_repos.txt ~/repos/./ '$remote:/tmp/repos/'", timeout => $timeout + 10, retry => 3, delay => 120);
+        script_retry("rsync --timeout=$timeout -vruahRd -e ssh --files-from /tmp/transfer_repos.txt ~/repos/./ '$remote:/tmp/repos/'", timeout => $timeout + 10, retry => 3, delay => 120);
```

Comment:
From the rsync manpage:

`--archive, -a            archive mode is -rlptgoD (no -A,-X,-U,-N,-H)`

Reply:
I think it's still good to make the `--recursive` explicit. But why turn verbosity on now?

Reply:
@ricardobranco777 correct, correct ;) But `man rsync` says more:
> --files-from=FILE
>    Using this option allows you to specify the exact list of files to transfer (as read from the specified FILE or '-' for standard input).  It also tweaks the default  behavior  of
>    rsync to make transferring just the specified files and directories easier:
> 
>      o      The  --relative  (-R)  option is implied, which preserves the path information that is specified for each item in the file (use --no-relative or --no-R if you want to turn that off).
> 
>      o      The --dirs (-d) option is implied, which will create directories specified in the list on the destination rather than noisily skipping them (use --no-dirs or --no-d if you want to turn that off).
> 
>      o      The --archive (-a) option's behavior does not imply --recursive (-r), so specify it explicitly, if you want it.
> 
>      o      These  side-effects  change the default state of rsync, so the position of the --files-from option on the command-line has no bearing on how other options are parsed (e.g. -a works the same before or after --files-from, as does --no-R and all other options).

Reply:
Why do we turn verbosity on now?

Reply:
It was just for debug, I disabled it again. Please re-check as I did some other changes and improved the documentation.

Reply:
ok